src/foam/u2/view/CurrencyView.js

```diff
@@ -9,7 +9,7 @@ foam.CLASS({
   name: 'CurrencyView',
   extends: 'foam.u2.view.ModeAltView',
 
-  documentation: 'A view for foam.core.Currency properties.',
+  documentation: 'A view for foam.core.UnitValue properties.',
```

Comment:
Should change the name of the View and places it's used as well then.

Reply:
I think this might need to stay as currency view. cmd-shift-R error.